docs/themes/godocs-2/assets/scss/templates/search-modal.scss

```diff
@@ -1,10 +1,53 @@
 // search-modal
-.search-modal-dialog {
-  margin-top: 150px;
+@media (min-width: 576px){
+  .modal {
+    --bs-modal-margin: 1.75rem;
```

Comment:
While trying to figure out why the modal's backdrop wasn't appearing, I compared the documentation site to one of the theme's live preview sites and noticed that some styles appeared to be missing. They didn't appear to be deleted in the past, so I'm not sure why the styles were removed. I copied the styles beginning with `--bs-modal` I found while using the inspector and that resolved the style issues I couldn't quite fix.